conf/openshift/openshift.json

```diff
@@ -142,7 +143,7 @@
         "name": "dataverse-plus-glassfish"
       },
       "spec": {
-        "dockerImageRepository": "iqss/dataverse-glassfish:latest"
+        "dockerImageRepository": "charlesthao96/dataverse-glassfish:4764"
```

Comment:
I take it this needs to be changed now that we are pulling in the code?

Reply:
that should be iqss/dataverse-glassfish. thanks for catching that